relay-sampling/src/lib.rs

```diff
@@ -745,23 +798,63 @@ pub struct DynamicSamplingContext {
     /// Set on transaction start, or via `scope.transaction`.
     #[serde(default)]
     pub transaction: Option<String>,
+    /// The sample rate with which this trace was sampled in the client. This is a float between 0 and
+    /// 1.
+    #[serde(
+        default,
+        with = "sample_rate_as_string",
+        skip_serializing_if = "Option::is_none"
+    )]
+    pub sample_rate: Option<f64>,
     /// The user specific identifier ( e.g. a user segment, or similar created by the SDK
     /// from the user object).
     #[serde(flatten, default)]
     pub user: TraceUserContext,
+    /// Spillover values for backwards/forwards compat
```

Comment:
Our usual doc comment for this is:

```suggestion
    /// Additional arbitrary fields for forwards compatibility.
```

Reply:
done